internal/core/application/operator_service.go

```diff
@@ -1315,7 +1315,40 @@ func (o *operatorService) MarketFragmenterSplitFunds(
 		return
 	}
 
-	o.splitMarketFragmenterFunds(ctx, mkt, utxos, int(millisatsPerByte), chRes)
+	chRes <- FragmenterSplitFundsReply{
+		Msg: "fetching fee account funds",
+	}
+	feeUtxos, err := o.getAllUnspentsForAccount(ctx, domain.FeeAccount)
+	if err != nil {
+		errMsg := fmt.Errorf(
+			"error while fetching fee account funds: %s", err,
+		)
+		if err == domain.ErrVaultAccountNotFound {
```

Comment:
maybe warn also fee treshold?

Reply:
No. If the fee account is not found it means that no addresses have ever been generated. This means that the operator is trying to deposit into the market (via fragmenter) before funding the fee account.